.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
```

Comment:
Not adding https://github.com/dense-analysis/ale/blob/7e3d2930d8defbcb233b0bc21822cb8e8377bec2/.travis.yml#L9 from the original travis file?

Reply:
This was not working for some reason.  Not sure what is the exact syntax in Github actions to trigger on tags instead of branches.

Reply:
Yeah.  Just noticed that it already exists an image on Docker Hub.

Reply:
Some example here: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags

Maybe just use:

```
on:
  push:
    tags:
      - /^v\d+\.\d+\.(x|\d+)$/ 
      ```

Regex syntax: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet